.eslintrc

```diff
@@ -253,6 +256,91 @@
     "space-infix-ops": 2, // http://eslint.org/docs/rules/space-infix-ops
     "space-in-parens": [2, "never"], // http://eslint.org/docs/rules/space-in-parens
     "spaced-comment": [2, "always"], // http://eslint.org/docs/rules/spaced-comment
-    "arrow-spacing": [2, { "before": true, "after": true }] // https://eslint.org/docs/rules/arrow-spacing
+    "arrow-spacing": [2, { "before": true, "after": true }], // https://eslint.org/docs/rules/arrow-spacing
+
+    /**
+    * Additional rules to enable one by one
+    */
+    // "array-bracket-spacing": ["error", "never"],
+    // "array-callback-return": ["error", { "allowImplicit": true }],
+    // "arrow-body-style": ["error", "as-needed", { "requireReturnForObjectLiteral": false }],
+    // "arrow-parens": [ "error", "as-needed", { "requireForBlockBody": true }],
+    // "block-spacing": ["error", "always"],
+    // "computed-property-spacing": ["error", "never"],
+    // "dot-location": ["error", "property"],
+    // "function-paren-newline": ["error", "multiline"],
+    // "import/export": "error",
+    // "import/first": ["error", "absolute-first"],
+    // "import/newline-after-import": "error",
+    // "import/no-duplicates": "error",
+    // "import/no-mutable-exports": "error",
+    // "import/no-named-default": "error",
+    // "new-parens": "error",
+    // "newline-per-chained-call": ["error", { "ignoreChainWithDepth": 4 }],
```

Comment:
4 is sensible

Reply:
Just copied this one from Airbnb, so I'm ok with any number we agree on